src/sentry/incidents/logic.py

```diff
@@ -1446,3 +1474,14 @@ def rewrite_trigger_action_fields(action_data):
     if "settings" in action_data:
         action_data["sentry_app_config"] = action_data.pop("settings")
     return action_data
+
+
+def get_filtered_actions(validated_alert_rule_data, action_type: AlertRuleTriggerAction.Type):
```

Comment:
This code is totally fine as is but I thought you'd like to see it as:
```py
def get_filtered_actions(
    validated_alert_rule_data: Mapping[str, Any],
    action_type: AlertRuleTriggerAction.Type
) -> Sequence[Mapping[str, Any]]:
    from sentry.incidents.serializers import STRING_TO_ACTION_TYPE

    return [
        rewrite_trigger_action_fields(action)
        for trigger in validated_alert_rule_data.get("triggers", [])
        for action in trigger.get("actions", [])
        if STRING_TO_ACTION_TYPE.get(action.get("type")) == action_type
    ]
```

Reply:
Made the change, thanks!